README.md

```diff
@@ -42,4 +40,32 @@ The JVM will then listen for a debugger to be attached on port 8189.  The JVM wi
 the debugger to be attached before starting Echo; the relevant JVM arguments can be seen and
 modified as needed in `build.gradle`.
 
-[//]: # "Only here to retrigger the echo build"
+## Configuring
+echo can run in two modes: in-memory and SQL.
+
+
```

Comment:
extra newline. This is going to cause massive waste if people want to print this out.

Reply:
this is just the markdown way of specifying a new paragraph

Reply:
you only need one blank line, not two though

Reply:
fine, FINE. I will save some trees